environment.yml

```diff
@@ -33,6 +33,9 @@ dependencies:
   - pytest
   - pytest-cov
   - python-kaleido
+  - shap
+  - numba=0.55.0
```

Comment:
do we need the hard pin here?

Reply:
Let me see if I can do >=55.0 to get it working with Shap